client/src/helpers/APICalls/getUserBoards.ts

```diff
@@ -0,0 +1,17 @@
+import { FetchOptions } from '../../interface/FetchOptions';
+import { BoardApiData } from '../../interface/Board';
+
+const getUserBoards = async (): Promise<BoardApiData> => {
+  const fetchOptions: FetchOptions = {
+    method: 'GET',
+    headers: { 'Content-Type': 'application/json' },
+    credentials: 'include',
+  };
+  return await fetch(`/users/board`, fetchOptions)
+    .then((res) => res.json())
+    .catch((err) => ({
+      error: { error: err, message: 'Unable to connect to server. Please try again' },
+    }));
+};
+
+export default getUserBoards;
```

Comment:
Will there also be a single route like `getUserBoard` that only returns a single board?

Reply:
It is possible, especially if there is board sharing in the future.